modules/spo-base-syscalls.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="spo-base-syscalls_{context}"]
+= Base syscalls for a container runtime
+
+You can use `runc` to establish the minimum required syscalls for a given runtime to start a container.
```

Comment:
I think this was meant to say "You can use the `baseProfileName` attribute to establish the minimum..".
What the attribute does is that it merges the allowed syscalls from the `baseProfileName` with the syscall names that are explicitly allows through the `.spec.syscalls.names` list.

Actually, let me ask @saschagrunert - do we even support other runtimes than `runc` on OCP? Should we suggest that on OCP, all `SeccompProfiles` should inherit from `runc-v1.0.0`? 

Also, while we have the profile as upstream manifest, do you think we should automatically create the profile maybe in the SPO namespace so that it's easier to use for OCP customers?

Reply:
We're pretty close to tech preview of crun in OpenShift 4.12 and will consider moving forward with that. So I'd say we support both for now in the SPO.

Reply:
> We're pretty close to tech preview of crun in OpenShift 4.12 and will consider moving forward with that. So I'd say we support both for now in the SPO.
> 
> > Also, while we have the profile as upstream manifest, do you think we should automatically create the profile maybe in the SPO namespace so that it's easier to use for OCP customers?
> 
> Yeah, we should probably rework our default profiles. thinking

So one thing we could do would be to use `go:embed` to embed the `examples/baseprofile-runc.yaml` file so that it gets created automatically at least in the SPO namespace. Unsure how to deal with profiles in other namespaces though.

I'll file an upstream ticket so we can discuss this further. Thanks for chiming in Sascha!

Reply:
FWIW, https://github.com/kubernetes-sigs/security-profiles-operator/issues/1313